fact_extractor/plugins/unpacking/avm_kernel_image/test/test_avm_kernel.py

```diff
@@ -0,0 +1,15 @@
+from test.unit.unpacker.test_unpacker import TestUnpackerBase
+from ..code.avm_kernel_image import FIND_SQUASHFS_TOOL_PATH, UNPACK_KERNEL_TOOL_PATH
+
+
+class TestAvmKernelImage(TestUnpackerBase):
```

Comment:
I'd like to have a real test here obviously. But you say that's not possible?

Reply:
We could download and extract real firmware during installation. However, since the vendor does not provide a public archive, we would have to fix this test each time the vendor publishes a new firmware version.